gradle.properties

```diff
@@ -2,4 +2,4 @@
 
 # Use this property to select the most recent Brightcove Android
 # Native Player version.
-anpVersion=4.3+
```

Comment:
This change is fine for testing locally, but it seems like it should be removed from the PR.